bool start
---
bool result
---
float32 distance
bool module_lifted
